types/react-select/index.d.ts

```diff
@@ -178,6 +178,10 @@ declare namespace ReactSelectClass {
          * if true, a loading spinner will be shown at the right side.
          * @default false
          */
+         instanceId?: string;
```

Comment:
1. Can you provide a link to documentation (or a line of code) which proves that this exists and where it's set on the object?
2. You put the comment below the item, so now `isLoading` and `instanceId` have their comments swapped.
3. It looks like the indentation is off -- you have one extra space before `instanceId`.